components/x-teaser/src/Title.jsx

```diff
@@ -12,6 +12,7 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 			<Link {...props} url={displayUrl} attrs={{
 				'data-trackable': 'heading-link',
 				className: 'js-teaser-heading-link',
+				'aria-label': props.type === 'video' ? `Watch video ${displayTitle}` : null
```

Comment:
For podcasts/audio should we also add a "listen to" prefix?

Reply:
Yes 👍